### DISCUSSION TITLE
Getting error -> Method App\Http\Livewire\Users::deleting does not exist.

### DISCUSSION THREAD

**Discussion by soccerjf** [spatie/laravel-permission#1988] [Q&A]
I'm using spatie/laravel-permission v5.4.1
I'm using Laravel 8 with Livewire.  I'm attempting to use Spatie\Permission\Traits\HasRoles, so I add the following to my php script
      **_use Spatie\Permission\Traits\HasRoles;_**
and in the class component
      _**use HasRoles**_
when I run the script I now get the error 
    **_Method App\Http\Livewire\Users::deleting does not exist_**.
The script was working with the **_use Spatie\Permission\Models\Role_** and _**use Spatie\Permission\Models\Permission**_; added.
Any insightrs would be appreciated, thxs.

**Comment by erikn69** [comment]
Maybe related #1984
